images/linux/scripts/installers/php.sh

```diff
@@ -93,6 +93,10 @@ apt-get remove --purge -yq php7.2-dev
 
 apt-fast install -y --no-install-recommends snmp
 
+if isUbuntu20 ; then
+    php_versions="7.4"
+fi
```

Comment:
Why you copy-paste these lines from 27-29 lines of this file?

Reply:
For sake of further evolution. There will be ubuntu21 some day and thought it would be nice to have the suggestion for a developer it is possible to branch